pmm/aws-staging-start.groovy

```diff
@@ -73,7 +73,7 @@ pipeline {
             description: "Query Source for Monitoring",
             name: 'QUERY_SOURCE')
         text(
-            defaultValue: '-e METRICS_RETENTION=192h',
+            defaultValue: '',
```

Comment:
Why is this removed, we use it to verify default docker server behavior when users pass the wrong variable, it should only throw an error in the logs.

Reply:
Sorry, I didn't know that was part of our tests. However, it seams strange and misleading, because I initially took it for the face value and thought that this param is real! I even changed it to 30 days and then saw it wasn't working. 

So, what do we do @puneet0191? Can't we test for illegal env vars in our test pipelines? It seems like tests belong in there.

Reply:
I understand your point, I will modify the test suite job to handle this, I think we can remove it for now as the default value.